### DISCUSSION TITLE
v.0.3.0

### DISCUSSION THREAD

**Discussion by ivotaHR** [kubilus1/autoortho#43] [General]
Thank you for a new update.
I downloaded all world package.
Q: If we remove only folder 'z_autoortho', how recreate (make junction) 'z_autoortho' without re-downlaod all world packages, if that possible?

**Comment by kubilus1** [comment]
A couple of things!  Firstly, thanks for trying this out!   Secondly, this is currently a 'pre-release' so I'll likely tweak a few things.  Mostly minor I hope.  I'd love any feedback on what works and what doesn't.

This release will let you just download a binary and run it.  Simple as that.

As far as scenery packages and z_autoortho.  You should not have to do anything for those.  They should just continue to work as is.
